components/suggestion/switch_channel_provider.jsx

```diff
@@ -150,7 +150,10 @@ class SwitchChannelSuggestion extends Suggestion {
                 {...Suggestion.baseProps}
             >
                 {icon}
-                <span>{displayName}</span>
+                <span
+                    aria-label={displayName}
```

Comment:
Shouldn't the contents of the span be read instead of needing a label here?

Reply:
Maybe we could look at changing that instead? I think using the visible data for accessibility is preferred over adding the aria attributes whenever possible.

Reply:
@nevyangelova Where is the `aria-hidden` prop being added? I can't find it myself.

Reply:
Correction, I am not able to find a hidden prop either so in this case I cannot debug why its not working without adding the label.